.github/workflows/nodejs.yml

```diff
@@ -12,9 +12,9 @@ jobs:
         node-version: [14.x, 16.x]
```

Comment:
I think you should update this node version to 16 & 18. Then I think builds will work for both PRs

Reply:
This change is done.